src/data/java_profile.rs

```diff
@@ -126,15 +74,112 @@ impl CollectData for JavaProfileRaw {
                 }
             }
         }
-
         Ok(())
     }
 
-    fn collect_data(&mut self) -> Result<()> {
-        Ok(())
+    fn get_jid(&mut self, key: &str) -> Option<String> {
+        if self.process_map.contains_key(key) {
+            return Some(key.to_string());
+        }
+
+        self.process_map.iter().find_map(|(jid, name)| {
+            if key == name[0] {
+                Some(jid.to_string())
+            } else {
+                None
+            }
+        })
     }
 
-    fn finish_data_collection(&mut self, _params: CollectorParams) -> Result<()> {
+    fn update_process_map(&mut self) -> String {
+        info!("Running jps (may incur utilization spike)...");
+        let jps_cmd = Command::new("jps").output();
+        let mut jps_str = String::new();
+        match jps_cmd {
+            Ok(jps_out) => {
+                jps_str = String::from_utf8(jps_out.stdout).unwrap_or_default();
+                let jps: Vec<&str> = jps_str.split_whitespace().collect();
+                for i in (0..jps.len()).step_by(2) {
+                    if jps[i + 1] != "Jps" {
```

Comment:
Add a print for the new JVM with name and JID. Both this and the print at the start of the function should be debug.

Reply:
New JVM/JID info is printed at the trace level when async-profiler is successfully launched. Should that be changed to debug?